addon/data/shooter-interactive-worker.js

```diff
@@ -273,6 +281,9 @@ stateHandlers.crosshairs = {
   },
 
   mousedown: function (event) {
+    if (event.target.className === "pageshot-myshots") {
+      return;
+    }
```

Comment:
Will this work if you click on an element inside `.pageshot-myshots` (such as the `<span class="pageshot-myshots-text">...</span>`)?  The isHeader function below makes a more thorough check.